src/components/Input/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+
+import { InputSt } from './style';
+
+interface InputProps {
+  type: string;
+  idName: string;
+  placeholder: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const Input: FC<InputProps> = ({ type, idName, placeholder, onChange }) => {
+  return (
+    <InputSt
+      id={idName}
+      name={idName}
+      type={type}
+      placeholder={placeholder}
+      onChange={onChange}
+    ></InputSt>
```

Comment:
Закрывающий тег тут лишний

```suggestion
    />
```